python/tk_multi_workfiles/my_tasks/my_tasks_model.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, project, user, extra_display_fields, parent, bg_task_manager=
         """
         self.extra_display_fields = extra_display_fields or []
         filters = [["project", "is", project],
-                   ["task_assignees", "is", user]]
+                   ["task_assignees", "is", user],
```

Comment:
+1 for full control on this. The more control users have, and the happier they will be ;)

PS: Sorry to intervene in your PR, I don't know if we are allowed or not to do so...